dotnet/CONTRIBUTING.md

```diff
@@ -14,3 +14,9 @@ Alternatively, run `msbuild` from this directory.
 
 Keep in mind that this will only run unit tests. The acceptance tests are only
 run when you build with `make`.
+
+## Make a release
+
+
+    # Change version in `Gherkin.NuGetPackages\Gherkin.nuspec`
+    .nuget\NuGet.exe pack Gherkin.NuGetPackages\Gherkin.nuspec
```

Comment:
The release instructions should be for Linux/OS X, since that's what the release managers use (they will typically release all language implementations at the same time).

Therefore, please use forward slashes, and prefix the command with `mono --runtime=v4.0` as in the `Makefile`